app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  private
+
+  def f x
+    return [10, 20, 30].each { |i| i - x }.select { |i| i % 2 == 0 }.map {|i| "_#{i}_" }.each { |s| p s }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Space between { and | missing.